kratos/includes/registry.h

```diff
@@ -121,29 +122,60 @@ class Registry
     ///@name Access
     ///@{
 
+    static auto begin()
+    {
+        return mspRootRegistryItem->begin();
+    }
+
+    static auto cbegin()
+    {
+        return mspRootRegistryItem->cbegin();
+    }
+
+    static auto end()
+    {
+        return mspRootRegistryItem->end();
+    }
+
+    static auto const cend()
+    {
+        return mspRootRegistryItem->cend();
+    }
+
     static RegistryItem& GetItem(std::string const& rItemFullName);
 
+    template<typename TDataType>
+    static TDataType const& GetValue(std::string const& rItemFullName)
+    {
+        return GetItem(rItemFullName).GetValue<TDataType>();
+    }
 
     static void RemoveItem(std::string const& ItemName);
 
     ///@}
     ///@name Inquiry
     ///@{
 
+    static std::size_t size();
```

Comment:
Should be Size?, for consistency with other classes. size is actually more STL

Reply:
I did it like this because at the end this is a sort of extension of an stl unordered map. I don't mind changing it. The one people likes the most will be fine for me.